pkg/generate/app/cmd/newapp_test.go

```diff
@@ -53,6 +53,17 @@ func TestValidate(t *testing.T) {
 			env:                 map[string]string{"one": "first", "two": "second", "three": "third"},
 			parms:               map[string]string{},
 		},
+		"envfile": {
+			cfg: AppConfig{
+				GenerationInputs: GenerationInputs{
+					EnvironmentFile: "/tmp/envfile1",
+				},
+			},
+			componentValues:     []string{},
+			sourceRepoLocations: []string{},
+			env:                 map[string]string{"one": "first", "two": "second", "three": "third"},
+			parms:               map[string]string{},
+		},
```

Comment:
you're right i missed the call to cfg.validate(), but where did /tmp/envfile1 come from?  you're not creating it in this test.  how is it created and what content does it have?